add-on/src/recovery/recovery.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+/* eslint-env browser, webextensions */
+
+import choo from 'choo'
+import html from 'choo/html/index.js'
+import browser, { i18n, runtime } from 'webextension-polyfill'
+import { nodeOffSvg } from '../landing-pages/welcome/page.js'
+import createWelcomePageStore from '../landing-pages/welcome/store.js'
+import { optionsPage } from '../lib/constants.js'
+import './recovery.css'
+
+const app = choo()
+
+const learnMoreLink = html`<a class="no-underline" href="https://docs.ipfs.tech/how-to/companion-node-types/" target="_blank" rel="noopener noreferrer">${i18n.getMessage('recovery_page_learn_more')}</a>`
```

Comment:
(reusing link style from welcome page)
```suggestion
const learnMoreLink = html`<a class="navy link underline-under hover-aqua" href="https://docs.ipfs.tech/how-to/companion-node-types/" target="_blank" rel="noopener noreferrer">${i18n.getMessage('recovery_page_learn_more')}</a>`
```